### ISSUE TITLE
Hundreds of Villagers spawn

### ISSUE THREAD

**Issue by Kherab** [Fevix/Avali#187]
This may be a few separate issues and one of them might not be related to your mod. 

I am running a small dedicated server with a few mods including this one. An avali npc town spawned, on the planet we set up our colony. Visiting it the next (RL) day it now has hundreds of avali in what looks like a central square causing massive lag whenever we go near it.  When near it the server throws the following error repeatedly.

[Error] Could not instantiate item '[avaliteir3minigin, 1, {}]' (ItemException) No such item 'avalitier3minigin'

And also several of the error

[Error] Could not apply patch from source: ..\..\..\workshop\content\211820\729558042\contents.pak. Caused by: (JsonPatchException) Could not apply operation to base. (TraversalException) No such key 'helpme' in pathApply("/helpme/avali")

**Comment by Fevix** [comment]
Latter is known, first is probably a typo somewhere in a file. Looking and fixing...

**Comment by Kherab** [comment]
This evening I tried visiting the location again after starting starbound normally a few times to make sure it fetched updates. When I arrived I got a bunch of the "minigin" errors as I approached. About a dozen or so of the NPCs also seemed to die upon first arrival.

I also was able to notice some additional details and put a few things together. Some of the NPCs have matter manipulators out, and are using them on squares below the bridges. They were definitely doing that last night, I just did not put the pieces together.  I also got some much longer errors this time, too long and complicated for me to transcribe so I found my log file which is attached.

I also tried pullling out all the avali stuff in the problem area, both in the foreground and background. It did not cause any of them to despawn, or stop the errors.

[starbound_server.log.txt](https://github.com/Fevix/Avali/files/385208/starbound_server.log.txt)
